src/NFTBackedToken.sol

```diff
@@ -53,7 +57,7 @@ contract NFTBackedToken is
     ) public initializer {
         __Ownable_init(owner_);
         __ERC20_init(name_, symbol_);
-        __ERC20Permit_init(name_);
+        __ERC20Permit_init(name_); // do we want to add some test for the permit? or no need?
```

Comment:
@davidbrai IMO we don't need to test permit, trusting OZ, esp since we haven't made a single change to the underlying ERC20 logic. if you have concerns let's discuss asap?

Reply:
we definitely don't need to test all the use cases, but maybe just one happy flow test to see it works? i.e. to make sure we're inheriting & initializing correctly

Reply:
added a test, resolving this thread